doc/src/arch/reference.rst

```diff
@@ -1897,6 +1902,98 @@ The clocking configuration is specified with ``<clock>`` tags within the ``<cloc
     :opt_param C_wire_per_m: The wire capacitance, in Farads per Meter.
     :opt_param buffer_size: The size of each clock buffer.
 
+**Specifing a Clock Architecture**
```

Comment:
This doesn't talk about the virtual clock sink, and how that is placed within the grid?

Reply:
My understanding is that it's specified in the `clocknetworks/clock_routing/tap[@from="ROUTING"]` element. See `setup_clock_connections()` in `setup_clocks.cpp`.

Reply:
Note that the current code seems to only support one virtual clock network root, with the index stored in the `DeviceContext`.